packages/apps-v5/test/commands/ps/index.js

```diff
@@ -12,18 +12,18 @@ const hourAgoStr = strftime('%Y/%m/%d %H:%M:%S %z', hourAgo)
 
 function stubAccountQuota (code, body) {
   nock('https://api.heroku.com:443')
-    .get('/account/features/free-2016')
+    .get('/account/features/eco-2022')
```

Comment:
Good catch. I thought the feature name was innocuous naming in the test and since we won't support free, I was renaming it. That was a mistake on my part. Thanks for the catch!

Reply:
Ok cool, if that didn't break the tests I'm guessing the code is not even making that request to check that feature flag anymore (it is super old!), so maybe we can just remove this stub.